keylime/cmd/ima_emulator_adapter.py

```diff
@@ -4,11 +4,11 @@
 import select
 import sys
 import time
-from typing import Optional
+from typing import List
```

Comment:
Seems that this should be kept as Optional

Reply:
Fixed.